src/commonmark/en/content/implementation/localization-of-dhis2.md

```diff
@@ -27,96 +31,96 @@ In French the same key/value pairs would appear as follows
->     simply leave it blank. By default, DHIS2 will use English values
+>     simply leave it blank. By default, DHIS 2 will use English values
 >     for all strings which have not been translated.
 > 
 >   - All translated strings must be stored in escaped UTF-8 format. If
->     you are using the translation portal, be sure your browser
->     settings are set to UTF-8 when translating. If you are using a
->     text editor or other tool such as an IDE, you may need to convert
+>     you are using the DHIS 2 translation portal (discussed below), be sure 
+>     your browser settings are set to UTF-8 when translating. If you are using
+>     a text editor or other tool such as an IDE, you may need to convert
 >     the UTF-8 characters to escaped syntax, using the Java
->     "native2ascii" utility.
+>     `native2ascii` utility.
 > 
->   - Some special variables (e.g. {0} ) used curly brackets. This
+>   - Some special variables (e.g. {0} ) use curly brackets. This
 >     denotes a variable which will be replaced by a number or other
 >     value by the application. You must place this variable notation in
->     the correct position.
+>     the correct position and be sure not to modify it. 
 
 There are a number of tools which can be used to support the
 localization of the user interface as well as the database content,
 which will be discussed below.
 
-## DHIS2 i18n tool
+## DHIS 2 i18n tool
 
 <!--DHIS2-SECTION-ID:dhis2-i18n-translations-tool-->
 
-The i18n resource editor is Java desktop application which can be
-downloaded from http://www.dhis2.org/downloads . It requires that you
-have checked out the DHIS2 source code from Github (check out
-http://www.dhis2.org/development if necessary) and have a Java Runtime
-Environment installed on your computer. On Windows, simly unpack the ZIP
+The i18n resource editor is Java desktop application which can beYour translations
+downloaded from http://www.dhis2.org/downloads . It requires thatwill be incorporated into the source code on a regular basis by the
+have checked out the DHIS 2 source code from GitHub (check out
+http://www.dhis2.org/development if necessary) and have a Java Rudevelopment team.e
+Environment installed on your computer. On Windows, simply unpack the ZIP
 archive and click the executable file. On Linux, extract the archive,
 navigate inside the extracted directory and invoke the following:
 
     java -jar dhis-i18n-resourceeditor.jar
 
 1.  Press "Browse" when the application starts and select the path to
     the "dhis-2" directory inside your local copy (checkout) of the
-    DHIS2 source code repository, followed by OK.
+    DHIS 2 source code repository, followed by OK.
     
     
     ![](resources/images/i18n/i18n_tool_startup.png)
-
-2.  Next, select the destination locale which you will translate strings
-    into. Remember that if you want to create or add to a general
-    language translation, select e.g. "Portuguese - \[pt\]. If you want
+Your translations
```

Comment:
This sentence repeats four times! I think it shouldn't be here at all - instead you should have a blank line, then the next numbered item.

Reply:
Got it. Will look at it.